doxygen/contributor_help_pages/adding_new_distributions.md

```diff
@@ -0,0 +1,473 @@
+## Adding A New Distribution {#new_distribution}
+
+Before reading this section it's a good idea to at least
+skim over the [getting started guide](@ref getting_started). Stan's univariate
+distribution functions must work with mixes of scalars and vectors. This
+requirement can make the code for the distributions look a bit daunting, but
+in the below we'll cover each of the general steps needed to add a new distribution.
+
+1. Get the distribution working in the Stan language.
+2. Write out the partial derivatives for each input
+3. Writing the function
+4. Testing the function
+
+
+### Get the Distribution Working In Stan
+
+We will use the normal distribution as an example and adding the lpdf
+function, though note for acceptance into Stan math a function must have
+it's respective `lpdf`, `lcdf`, `cdf`, `lccdf` and `rng` implimented.
```

Comment:
it's -> its. Same in some other cases throughout this text.
A typo in "implemented".